dagger/main.go

```diff
@@ -213,15 +267,14 @@ func (m *Pipely) Debug(ctx context.Context) *dagger.Container {
 		WithExec([]string{"go", "install", "github.com/fabio42/sasqwatch@8564c29ceaa03d5211b8b6d7a3012f9acf691fd1"}).
 		File("/go/bin/sasqwatch")
 
-	// github.com/xxxserxxx/gotop
+	// https://github.com/xxxserxxx/gotop
 	gotop := m.Golang.
 		WithExec([]string{"go", "install", "github.com/xxxserxxx/gotop/v4/cmd/gotop@bba42d08624edee8e339ac98c1a9c46810414f78"}).
 		File("/go/bin/gotop")
 
 	p, _ := dag.DefaultPlatform(ctx)
 	platform := platforms.MustParse(string(p))
-	// https://github.com/hatoo/oha/releases
-	oha := dag.HTTP("https://github.com/hatoo/oha/releases/download/v1.8.0/oha-linux-" + platform.Architecture)
+	oha := dag.HTTP("https://github.com/hatoo/oha/releases/download/v" + ohaVersion + "/oha-linux-" + platform.Architecture)
```

Comment:
Another dynamically resolved resource that might be good to pin to a hash.

Reply:
Indeed! Since this is a debugging component, meaning that we only use it locally & never ship it, I am less concerned about this component. Nice to have, for sure, but OK as is **for now**.